component/src/main/java/com/iluwatar/component/objects/Graphics.java

```diff
@@ -0,0 +1,9 @@
+package com.iluwatar.component.objects;
+
+public class Graphics {
+
+	public void draw(Sprite sprite, int xCoord, int yCoord) {
+		System.out.println("Drawing game object with cooredinates : DX " + sprite.getDx() + "DY :" + sprite.getDy()
+				+ " X :" + xCoord + " Y: " + yCoord);
```

Comment:
Use logger instead printing to standard output. See for example here https://github.com/iluwatar/java-design-patterns/blob/master/adapter/src/main/java/com/iluwatar/adapter/FishingBoat.java